app/controllers/scenario_imports_controller.rb

```diff
@@ -7,11 +7,7 @@ def create
     @scenario_import = ScenarioImport.new(params[:scenario_import])
     @scenario_import.set_user(current_user)
 
-    if @scenario_import.will_request_local?(scenarios_url)
-      render :text => 'Sorry, you cannot import a Scenario by URL from your own Huginn server.' and return
```

Comment:
I don't know the history, but If the guids at one time were expected to be unique across an entire instance of Huginn, then it makes perfect sense.